airbyte-protocol/models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -236,6 +236,12 @@ definitions:
         description: If the connector supports incremental mode or not.
         type: boolean
       # Destination Connectors Properties
+      supportsNormalization:
+        description: If the connector supports normalization or not.
```

Comment:
Can we add a comment:

    Normalization is currently implemented using dbt so it requires `supportsDBT` to be true for this to be true.